pkgs/applications/version-management/gex/default.nix

```
@@ -8,24 +8,25 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "gex";
-  version = "0.4.0";
+  version = "0.5.0";
 
   src = fetchFromGitHub {
     owner = "Piturnah";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-eRforLvRVSrFWnI5UZEWr1L4UM3ABjAIvui1E1hzk5s=";
+    hash = "sha256-//sQ0s8bBQzuu5aO3RjPRjFuVYiGW6BwSPoCWKAx9DQ=";
   };
 
   nativeBuildInputs = [ pkg-config ];
   buildInputs = [ libgit2 ];
 
-  cargoHash = "sha256-OEaNERozmJL8gYe33V/m4HQNHi2I4KHpI6PTwFQkPSs=";
+  cargoHash = "sha256-rkhkFnRDtMTWFM+E5C4jR7TWtHdy3WUtIzvGDDLHqtE=";
 
   meta = with lib; {
     description = "Git Explorer: cross-platform git workflow improvement tool inspired by Magit";
     homepage = "https://github.com/Piturnah/gex";
+    changelog = "https://github.com/Piturnah/gex/releases/tag/${src.rev}";
     license = with licenses; [ asl20 /* or */ mit ];
-    maintainers = with maintainers; [ azd325 Br1ght0ne ];
+    maintainers = with maintainers; [ azd325 Br1ght0ne evanrichter ];
```

```suggestion
    maintainers = with maintainers; [ azd325 evanrichter ];
```
might as well, since all the other changes were done by you two

---

Maybe we can add Piturnah then here directly too. Then we can also close this PR https://github.com/NixOS/nixpkgs/pull/241326

---

@Piturnah

---

how would I do that? add their PR branch as a remote, then cherry pick two commits (maintainers.nix + this line), then fix the merge conflict on this line?

---

@evanrichter you can probably just apply the changes directly rather than fiddling with cherry picking, but if it's inconvenient I don't mind just squashing the commits on my PR. Just hadn't got round to it yet :)

---

ah ok that doesn't look too bad :)

> [Piturnah](https://github.com/evanrichter/nixpkgs/commits?author=Piturnah) authored and [evanrichter](https://github.com/NixOS/nixpkgs/commits?author=evanrichter) committed 1 minute ago

---

@SuperSandro2000 I think this is resolved now!